src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
 public class Application {
+    public static final int countOfBalls = 3;
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Ball[] randomBalls = new Ball[countOfBalls];
```

Comment:
we can do without this variable and immediately specify the desired length in the loop